test/test.dart

```diff
@@ -194,4 +196,34 @@ Future<void> setupBaas() async {
   final client = await (cluster == null ? BaasClient.docker(baasUrl) : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!));
 
   baasApps.addAll(await client.getOrCreateApps());
-}
\ No newline at end of file
+}
+
+@isTest
+Future<void> syncTest(
+  String name,
+  FutureOr<void> Function(ApplicationConfiguration configuration) testFunction, {
+  String appName = 'flexible',
+  dynamic skip,
```

Comment:
I know `test` has `skip` as `dynamic`, but why repeat that mistake. Let us make it a `bool`.

Reply:
it can be a bool or a String that's why it is dynamic

Reply:
How would you call skip with a string?

Reply:
you pass a string in the skip arg. This is how the test package works